mdtraj/geometry/alignment.py

```diff
@@ -269,3 +269,33 @@ def rmsd_qcp(conformation1, conformation2):
     max_eigenvalue = scipy.optimize.newton(f, E0, df)
     rmsd = np.sqrt(np.abs(2.0 * (E0 - max_eigenvalue) / n_atoms))
     return rmsd
+
+def compute_average_structure(xyz):
+    """Compute the average structure from a set of frames.
+
+    The frames are first aligned to minimize the RMSD from the average structure,
```

Comment:
Why not add an optional argument to allow users to specify whether or not the frames should first be RMSD-aligned? I could imagine both schemes could be useful.